fil_fungible_token/src/receiver/types.rs

```diff
@@ -12,11 +13,14 @@ pub trait FRC46TokenReceiver {
     /// the receiving actor can immediately utilise the received funds. If the receiver wishes to
     /// reject the incoming transfer, this function should abort which will cause the token actor
     /// to rollback the transaction.
-    fn tokens_received(params: TokensReceivedParams);
+    fn receive(token_type: TokenType, params: FRC46TokenReceived);
 }
 
+pub type TokenType = u32;
```

Comment:
Change to `ReceiverType` or similar. The universal receiver isn't limited to "tokens".